akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/LocalReceptionist.scala

```diff
@@ -60,12 +96,12 @@ private[akka] object LocalReceptionist extends ReceptionistBehaviorProvider {
    * @param subscriptionsPerActor current subscriptions per subscriber (needed since a subscriber can subscribe to several keys) FIXME is it really needed?
    */
   private final case class State(
-      services: TypedMultiMap[AbstractServiceKey, Service],
+      services: ServiceMultiMap[ServiceF],
```

Comment:
Since this is private this change is OK, but our tooling that checks binary compatibility (mima) does flag it for review. Could you add an exclusion (e.g. `ProblemFilters.exclude[Problem]("akka.actor.typed.internal.receptionist.LocalReceptionist*`? You can run `sbt mimaReportBinaryIssues` to find more warnings, and CONTRIBUTING.md has more information on how/where to add the filters.

Reply:
done